atlp-rwanda/commanders-rn-medica
# Feat: add password reset screens

#### What does this PR do?

This PR includes designs of the forgot & reset password screens

#### Description of Task to be completed?

- Add reset password, otp verification, and new password screens
- Add navigation header component
- Add icon component
- Add typed routes configuration
- Add necessary props on button component
- Add input component

#### How should this be manually tested?

On the first page there is a link with text `Reset Password`, click it and it will redirect you to the reset password screen. Follow the continue buttons to check all screens.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/d06aee63-56a0-4f8b-96a1-cc3ba897b5e1" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/a27fe811-4469-4c6d-a0f1-841bbd892a3c" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/3046d6f1-9724-44bf-8a66-4f9491c60779" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/5989fe0f-a34d-44d2-ad5f-f5a8213bfdab" width="150" />
<img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/eb56408f-2bef-4720-a0ab-bbf6e7b19735" width="150" />

#### Questions:

N/A
